analytics_data_api/utils.py

```diff
@@ -98,3 +53,10 @@ def dictfetchall(cursor):
         dict(zip([col[0] for col in desc], row))
         for row in cursor.fetchall()
     ]
+
+
+def class_for_name(path):
+    """ Returns the class given the full path. """
```

Comment:
I see.  I'll update this to `load_fully_qualified_definition` and rename the variable to `definition`.